### ISSUE TITLE
Bio spreadsheets: (near) finalization in principle

### ISSUE THREAD

**Issue by SusanBrown** [cwrc/ontology#384]
I think we are pretty solid on what we're doing with the extractions for bio tags generally, with the exceptions of the following decisions:
1) whether we have classes like Birth, Death, Education--may be useful for subclassing e.g. DeathEvent, DeathContext, CauseofDeath, but we're waiting on ULO decision
2) whether we have a predicate like hasDeath linking the subject to their DeathContext
3) what we're doing with events, also waiting on ULO decision

However, I think we can move ahead with the extraction scripts without those decisions for the moment, so as to make progress generally, and go back and add the missing components when we've reached those decisions. If you disagree, please say so @joelacummings @DeborahStacey. 

If this is ok, I'd like to push ahead on reviewing and getting approval from Joel for several of the other spreadsheets, so that @alliyya and @GurjapSingh (who have pretty much finished CF and Family) can continue with the extraction scripting.

**Comment by joelacummings** [comment]
I agree I think we should try and progress with what's available now as I think as extraction is done it will shed light on other areas we may have missed or need more thought. Scripts can always tweaked and run again.
